x-pack/plugins/cases/common/constants/owners.ts

```diff
@@ -32,19 +34,31 @@ export const OWNER_INFO: Record<Owner, RouteInfo> = {
     label: 'Security',
     iconType: 'logoSecurity',
     appRoute: '/app/security',
+    validRuleConsumers: [AlertConsumers.SIEM],
   },
   [OBSERVABILITY_OWNER]: {
     id: OBSERVABILITY_OWNER,
     appId: 'observability-overview',
     label: 'Observability',
     iconType: 'logoObservability',
     appRoute: '/app/observability',
+    validRuleConsumers: [
+      // only valid in serverless
```

Comment:
Should there be separate list of validRuleConsumers for `serverless` and `ESS`?

Reply:
In both deployments, a valid consumer will be found in the `validRuleConsumers` array so there is no need to separate the list. If we separate it into two lists we will have to add logic to check if we are in `serverless` or `ESS` which will increase the complexity of the code.